facebook-instant-articles.php

```diff
@@ -386,6 +388,9 @@ function inject_ia_markup_meta_tag() {
 	}
 	add_action( 'wp_head', 'inject_ia_markup_meta_tag' );
 
+	// Injects the <link rel...> tag if the AMP Markup is enabled
+	add_action( 'wp_head', array('Instant_Articles_Amp_Markup', 'inject_link_rel') );
```

Comment:
As we discussed offline, we agreed to have all hooks explicit in the facebook-instant-articles.php so we can have a centralized view of all the touch points the plugin has with WP.